src/classes/Message.js

```diff
@@ -301,4 +302,47 @@ export default class Message {
       }
     }
   }
+
+  replaceEmojiText() {
+    const emojisDisabled = !config.get("enableEmojis");
+    if (emojisDisabled || !this.type === "message" || !this.body) {
+      return;
+    }
+
+    let { body, html } = this.body;
+    if (!body) {
+      return;
+    }
+
+    const regex = new RegExp(
```

Comment:
i should move this out of the function - i'll do that tomorrow when i'm less tired